source/Ship.h

```diff
@@ -310,6 +310,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	int Crew() const;
 	int RequiredCrew() const;
 	void AddCrew(int count);
+	// Get this ship's crew equivalent value.
+	const int CrewValue() const;
```

Comment:
We don't need to both with returning a const value.
```suggestion
	int CrewValue() const;
```

Reply:
oh I'm sorry you just checked all my code I was rewriting crew equivalent it to be stocked as an attribute and tested it like a minute ago

Reply:
> still wondering : do I need to go into outfit and say that crew equivalent can have any value ?

If you do then CrewValue needs to return `max<int>(0, max(Crew(), RequiredCrew()) + attributes.Get("crew equivalent"))` so that the CrewValue is never negative.

Reply:
I made it be 0 minimum don't want to have such risks 
governmnent that wants you to some of it's ships ? can't find a reason for it

Reply:
> I made it be 0 minimum

No, you made it so that "crew equivalent" can have any value, positive or negative.

From the comment above MINIMUM_OVERRIDE in Outfit:
> // Attributes which are present and **map to zero** may have **any value.**

The default behavior is that attributes can't be negative, but by adding "crew equivalent" to the MINIMUM_OVERRIDE, it can be negative.

Reply:
let's leave it as open as possible for content creators if you don't think it will break anything
also assisting ships gives a bonus rep so we can't put 0 as min to crewvalue